Fix the build with CMake 3.4.0.

Not upstreamable, kdeartwork is unmaintained.
--- kscreensaver/xsavers/CMakeLists.txt.orig	2015-11-26 12:38:07 UTC
+++ kscreensaver/xsavers/CMakeLists.txt
@@ -1,3 +1,4 @@
+include(CheckIncludeFiles)
 check_include_files(memory.h HAVE_MEMORY_H)
 configure_file (config-xsavers.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/config-xsavers.h )
 
